python/lsst/daf/butler/registry/_registry.py

```diff
@@ -203,19 +204,19 @@ def fromConfig(
         writeable: bool = True,
         defaults: Optional[RegistryDefaults] = None,
     ) -> Registry:
-        """Create `Registry` subclass instance from `config`.
+        """Create `Registry` subclass instance from ``config``.
 
         Registry database must be initialized prior to calling this method.
 
         Parameters
         ----------
         config : `ButlerConfig`, `RegistryConfig`, `Config` or `str`
             Registry configuration
-        butlerRoot : `lsst.resources.ResourcePathExpression`, optional
+        butlerRoot : convertible to `lsst.resources.ResourcePath`, optional
```

Comment:
Yes, there are, but [the link can't work](https://pipelines.lsst.io/v/d_2023_06_05/py-api/lsst.daf.butler.registry.Registry.html#lsst.daf.butler.registry.Registry.fromConfig) because `ResourcePathExpression` is not a documented type: . The same goes for any other Mypy-helper types.

It's not obvious to me, at least, what does or doesn't count as a "ResourcePath expression", so this is one case where I think it makes sense for the human-readable description to deviate from the machine-readable one. I'll admit `DatasetId` is a case where the meaning should be clear without explanation.

Reply:
Interesting: it looks like there are *two* `Registry` pages, https://pipelines.lsst.io/v/d_2023_06_05/py-api/lsst.daf.butler.registry.Registry.html and https://pipelines.lsst.io/v/d_2023_06_05/py-api/lsst.daf.butler.Registry.html, and the links work on one and are broken on the other. A good argument for avoiding unqualified links. 😬

Reply:
Yes, this is one of the things that Sphinx complains about.

> docstring of lsst.daf.butler.registry._registry.Registry:1: WARNING: duplicate object description of lsst.daf.butler.registry._registry.Registry, other instance in py-api/lsst.daf.butler.Registry, use :noindex: for one of them